prow/config/clustertrustbundles.yaml

```diff
@@ -0,0 +1,34 @@
+# This configs KinD to spin up a k8s cluster with ClusterTrustBundle support enabled
+# This should only be used to create K8s clusters with versions >= 1.27
+kind: Cluster
```

Comment:
Where is this Kind config used?

Reply:
not by anything in CI - it's only there for manual testing right now. We can use it when we add an integration test for ClusterTrustBundle

Reply:
Got it, thanks @dgn